README.md

```diff
@@ -31,6 +31,10 @@ for boot device and rootfs device respectively. In order to run this demo, the f
    NVMe drive 16GB minimum (recommended, faster);
    - USB flash drive 16GB minimum.
 
+It is important to ensure that selected either NVMe SSD or Flash USB dirve is clean and **does not contain any boot
+images** so that system will boot from SD card. Also make sure to backup the SD card image because the card will be
+overwritten during the deployment.
```

Comment:
We should add link to the official document on how to change boot order in this case or how to cleanup NVMe.

Reply:
Can you provide the links?

Reply:
https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#raspberry-pi-bootloader-configuration

Reply:
As agreed, cleaning out SSD / USB is out of scope of our documentation